tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -216,6 +258,22 @@ def _serialize_item(self, key: Literal[runstate.JOB_STATE, runstate.JOB_RUN_STAT
             prom_metrics.json_serialization_errors_counter.inc()
             return None
 
+    def _deserialize_item(self, key: str, state: str) -> Optional[Dict[str, Any]]:
```

Comment:
non-blocking nit: i think that we'll never hit the Optional case - all paths raise or return a dict according to the types :)
```suggestion
    def _deserialize_item(self, key: str, state: str) ->Dict[str, Any]:
```